apps/contracts/project.json

```diff
@@ -8,7 +8,7 @@
             "options": {
                 "commands": [
                     "hardhat node",
-                    "hardhat compile && hardhat deploy --network localhost"
+                    "hardhat node hardhat compile && hardhat deploy-zkgroups --network localhost"
```

Comment:
@saleel I noticed that this doesn't work on my side. The second command starts before the first one.

Reply:
@cedoor In this case we can add a delay before the second command. I had this initially but was working without as well for me.
Try this
```sleep 3 && hardhat compile && hardhat deploy-zkgroups --network localhost```

I don't understand your current change though - adding additional `hardhat node` before the second command?

Reply:
Ops, just fixed! Thanks